src/dataProcessor/dataProcessorService.js

```diff
@@ -1 +1,46 @@
+let getDebt = (
```

Comment:
why are we using `let` for defining methods here? will the `getDebt` variable be changed ? can we use `const`? Or should we use something else? how should we structure these functions?